BaSP-a2022/marta-b-trackgenix-app
# BaSP-TG-97: login firebase functionality in FE

- Create login form
- Create onSubmit function with roles using firebase
- Logout Redirect

**How to test:** 
- You can login as an employee, administrator or super administrator. 
- If you login as an administrator, you should be redirect to a page where you can see all employees, projects, timesheets and tasks. You can edit, add or delete this entities.
- If you login as an employee, you can see your profile and you can edit it, you also can see your timeesheets information and your projects information.
- If you login as a super adminstrator you can add, edit and delete admins. (Remember that a super admin is created from the backend).

**Credentials:**
- Employee:
  User: empleadazo@gmail.com 
  Password: empleadazo
- Admin
   User: nuevoadmin@gmail.com 
   Password: nuevoadmin
- Super Admin
   User: superadmin@gmail.com 
   Password: superadmin

**This things aren't finished yet:**
- Home page
- Unify styles - mobile first
- When you want to edit a profile it shouldn't let you change the password
- Admins sidebar should not show Tasks and Timesheets options

Contributors: @mara666 @PilarFernandezC @DamianPalavecino
